File: src/main/java/com/neutralplasma/virtusbot/commands/AudioCommand.java
package com.neutralplasma.virtusbot.commands;

import com.jagrosh.jdautilities.command.Command;
import net.dv8tion.jda.api.Permission;

public abstract class AudioCommand extends Command {

    public AudioCommand()
    {
        this.category = new Category("Music", event ->
        {
            return true;
            /*if(event.getAuthor().getId().equals(event.getClient().getOwnerId()))
                return true;
            if(event.getGuild()==null)
                return true;
            return event.getMember().hasPermission(Permission.MANAGE_SERVER);*/
        });
        this.guildOnly = true;
    }

}

File: src/main/java/com/neutralplasma/virtusbot/commands/AudioCommand.kt
package com.neutralplasma.virtusbot.commands

import com.jagrosh.jdautilities.command.Command
import com.jagrosh.jdautilities.command.CommandEvent
import com.neutralplasma.virtusbot.VirtusBot.blackList
import java.util.function.Predicate

abstract class AudioCommand : Command() {
    init {
        this.category = Category("Music", Predicate { event: CommandEvent ->
            !blackList.isBlackListed(event.author.id)
        })
        guildOnly = true
    }
}